tools/functional-tester/etcd-runner/command/lock_racer_command.go

```diff
@@ -69,12 +72,16 @@ func runRacerFunc(cmd *cobra.Command, args []string) {
 		m := concurrency.NewMutex(s, racers)
 		rcs[i].acquire = func() error { return m.Lock(ctx) }
 		rcs[i].validate = func() error {
+			mu.Lock()
```

Comment:
Maybe just tear our the entire doRounds acquire/release thing for elections. There's no way to sanely reason about this.